### PR TITLE
Added index.html

### PR DESCRIPTION
**Pull request by Louis-Clelland**
and eddited README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # test
 
 LCLE
+test
```
Comment (xlcle) [inline README.md:4]: new line?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>LCLE Home</title>
+</head>
+<body>
+  <h1>Louis Clelland</h1>
+  <h3>Website hosted on <a href="https://github.com">GitHub</a>, code from <a href="https://kevq.uk">Kev Quirk.</h3>
```
Comment (xlcle) [inline index.html:8]: missing </a> tag

File: README.md (hunk 1 above)
Comment (Louis-Clelland) [reply README.md:5]: sone

File: index.html (hunk 2 above)
Comment (Louis-Clelland) [reply index.html]: https://github.com/xlcle/test/pull/1/commits/d60c1ce98d9338072d0caa348c378ec52f1da016

**Pull request merged**